src/server/page_server.cpp

```diff
@@ -121,14 +121,18 @@ page_server::connection_handler::receive_log_page_fetch (tran_server_conn_t::seq
       _er_log_debug (ARG_FILE_LINE, "Received request for log from Transaction Server. Page ID: %lld \n", pageid);
     }
 
-  m_ps.get_page_fetcher ().fetch_log_page (pageid, std::bind (&page_server::connection_handler::on_log_page_read_result,
-      this, std::placeholders::_1, std::placeholders::_2));
+  auto callback_func = [this, sp = a_sp] (const LOG_PAGE *log_page, int error_code) mutable
```

Comment:
I checked and the compiler says that:
```
      sequenced_payload &operator= (const sequenced_payload &) = delete;
```
is ok. It only needs the copy ctor.

Reply:
Great, I'll remove both operator= and copy constructor in LETS-329. Neither should be necessary anymore.